src/store/reducers/authReducer.js

```diff
@@ -0,0 +1,31 @@
+const initialState = {
```

Comment:
I suggest having, an additional file called, initialState.js for all initial states. Easier maintenance. All initial states will be in same file. [Example](https://github.com/zero-to-mastery/mappypals/blob/dev/src/store/reducers/initialState.js)